test/utils.ts

```diff
@@ -120,7 +124,7 @@ export function getFakeBalanceProof(
   network: Network,
   nullifier: number,
   threshold: number
-) {
+): BalanceProofStruct {
```

Comment:
You shouldn't need to specify the return type if it's compatible with the struct